packages/backend/src/nest/storage/storage.service.ts

```diff
@@ -72,7 +72,7 @@ export class StorageService extends EventEmitter {
     private readonly localDbService: LocalDbService,
     private readonly orbitDbService: OrbitDb,
     private readonly certificatesRequestsStore: CertificatesRequestsStore,
-    private readonly certificatesStore: CertificatesStore,
+    public readonly certificatesStore: CertificatesStore,
```

Comment:
Making this public so we can access it from the registration service. We can also build wrapper functions in storage service and keep it private.

Reply:
Like getters and setters? Or wrapper functions for specific CertificatesStore functions, e.g. `StorageService.getCertificates` which is a wrapper for `CertificatesStore.getCertificates`?

Reply:
Either way is fine for me but I think getter/setter is a slightly better choice in this case.

Reply:
I went with the wrapper here, because that approach was already being used in this file